pkgs/applications/science/psychopy/default.nix

```diff
@@ -0,0 +1,92 @@
+{ lib
```

Comment:
psychopy is also, or even primarily, a Python library, not just a desktop application, so it should probably move to pkgs/development/python-modules (and use buildPythonPackage, which should clean things up a bit rather than e.g. bringing in `python310Packages`), and then an application package can be created in all-packages via `toPythonApplication`